app/views/studios/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1> Studio: <%= @studio.name %> </h1>
+Location: <%= @studio.location %> </br>
+
+Actors:
+<ul> <% @studio.distinct_actors.each do |actor| %>
+  <div id="actor-<%= actor.id %>">
+  Name: <%= actor.name %>, Age: <%= actor.age %> 
+  </div>
+<% end %> </ul>
```

Comment:
Please don't forget that this is an HTML page and all of the text in this file should be wrapped in some form of HTML.